README.md

```diff
@@ -1,2 +1,18 @@
-# interoperability-placeholder
-This is a placeholder for 2023 interoperability project 
+# Interoperability
+This project implements a Loss to Follow Up (LTFU) workflow system for CHIS based on [OpenHIE LTFU Guide](https://wiki.ohie.org/display/SUB/Use+Case+Summary:+Request+Community+Based+Follow-Up)
+
+## Get Started
+
+1. Run `./startup.sh init` to start-up the docker containers on the first run or after calling `./startup.sh destroy`. Use `./startup.sh up` for subsequent runs after calling `init` without calling `destroy`.
+
+1. The default User username for OpenHIM is `interop@openhim.org` and password is `interop-password`. The default Client username is `interop-client` and password is `interop-password`.
+
+1. `./startup.sh down` or `./startup.sh destroy` to shutdown the servers.
+
+1. Visit https://localhost:9000 and login with the following credentials email: `interop@openhim.org` and password: `interop-password`.
+
+1. Once logged in, visit https://localhost:9000/#!/mediators and select the only mediator with the `Name` 'Loss to Follow Up Mediator'.
+
+1. Select the green `+` button to the right of the default channel to add the mediator.
+
+1. You can test the mediator by running `curl -X GET http://localhost:5001/mediator -H "Authorization: Basic $(echo -n interop-client:interop-password | base64)"`. You should get the following response `{"status": "success" }`
```

Comment:
@samuelimoisili when running the curl command I'm getting:
`curl: (3) URL using bad/illegal format or missing URL`
And openhim-core logs shows:
```
2023-01-15 11:24:04 2023-01-15T14:24:04.404Z [worker1] info: starting job: generate transaction alerts, Last Ran at: Sun Jan 15 2023 14:24:04 GMT+0000 (Coordinated Universal Time)
2023-01-15 11:24:04 2023-01-15T14:24:04.424Z [worker1] info: Job generate transaction alerts has completed
2023-01-15 11:24:04 2023-01-15T14:24:04.429Z [worker1] info: starting job: auto retry failed transactions, Last Ran at: Sun Jan 15 2023 14:24:04 GMT+0000 (Coordinated Universal Time)
2023-01-15 11:24:04 2023-01-15T14:24:04.435Z [worker1] info: Job auto retry failed transactions has completed
2023-01-15 11:24:13 2023-01-15T14:24:13.233Z [worker1] info: No basic auth details supplied, trying next auth mechanism if any...
2023-01-15 11:24:13 2023-01-15T14:24:13.234Z [worker1] info: Could NOT authenticate via TLS: undefined, trying next auth mechanism if any...
2023-01-15 11:24:13 2023-01-15T14:24:13.241Z [worker1] info: The channel that matches the request /mediator is: Mediator
2023-01-15 11:24:13 2023-01-15T14:24:13.242Z [worker1] info: The request, '/mediator', is not authorised to access any channels.
```

Reply:
I also tried using the following command (generated by Postman): `curl -X GET -i http://localhost:5001/mediator -H 'authorization: Basic aW50ZXJvcC1jbGllbnQ6aW50ZXJvcC1wYXNzd29yZA==' -H 'cache-control: no-cache' -H 'postman-token: 0817aeb7-ea2d-f5d7-612b-38c6325a0b74'` and got: `HTTP 401 Unauthorized`
And openhim-core logs shows:
```
2023-01-15 12:25:58 2023-01-15T15:25:58.600Z [worker1] info: interop-client could NOT be authenticated, trying next auth mechanism if any...
2023-01-15 12:25:58 2023-01-15T15:25:58.601Z [worker1] info: Could NOT authenticate via TLS: undefined, trying next auth mechanism if any...
2023-01-15 12:25:58 2023-01-15T15:25:58.611Z [worker1] info: The channel that matches the request /mediator is: Mediator
2023-01-15 12:25:58 2023-01-15T15:25:58.611Z [worker1] info: The request, '/mediator', is not authorised to access any channels.
2023-01-15 12:26:03 2023-01-15T15:26:03.620Z [worker1] info: HTTP socket timeout reached
```